_scripts/deploy.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+#
+# Build and push Docker images to Docker Hub and quay.io.
+#
+
+if [ "$TRAVIS_PULL_REQUEST" != false ]; then
+    echo "Not deploying Docker images when TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST"
+    exit 0
+fi
```

Comment:
>Travis CI shouldn't deploy on pull requests (it already checks for this).

Good point--somehow I missed [that](https://docs.travis-ci.com/user/deployment/#Pull-Requests), since I was blindly following some logic in [helm](https://github.com/helm/helm/blob/master/.travis.yml#L37) that I assume is also redundant. I'll simplify this and the other related PRs, thanks!